app/services/idv/ssn_edit_distance_calculator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Idv
+  class SsnEditDistanceCalculator
+    attr_reader :previous_ssn, :current_ssn
+
+    # @param previous_ssn [String]
+    def initialize(previous_ssn, current_ssn)
+      @previous_ssn = previous_ssn
+      @current_ssn = current_ssn
+    end
+
+    def compute
+      previous_ssn.chars.zip(current_ssn.chars).reduce(0) do |acc, chars|
+        if chars.first == chars.last || chars.last.nil?
+          acc
+        else
+          acc + 1
+        end
+      end
+    end
```

Comment:
what if we used the DidYouMean gem (which is built-in to Ruby since 2.3), which has a [built-in Levenshein distance calculator](https://github.com/ruby/did_you_mean/blob/master/lib/did_you_mean/levenshtein.rb#L7)

Reply:
and honestly maybe we could remove this entire class and just inline it?
```suggestion
    def compute
      DidYouMean::Levenshtein.distance(previous_ssn, current_ssn)
    end
```

Reply:
Just caught the comment about "why not Leveinshtein" -- my b

Reply:
One more thought, I think Levenshtein would be the same result, with less code because it's from an already-included library! I think it would be worth considering using this built-in.

Reply:
I was pairing with @lmgeorge on this. I didn't realize this was built in! Yeah, I think we can go back and simplify this a good bit with the built in tooling.